src/app/client/src/app/modules/dashboard/components/course-progress-exhaust/data.ts

```diff
@@ -0,0 +1,182 @@
+export const courseProgressData = {
+
+    "result": {
```

Comment:
is that fille required? @sanjeev-scm

Reply:
It is required until I get the data from the API. Not adding this file resulted in error as there was import. I will remove this file once the API call is successful